Pocket/pocket-ios
# Shared With You - Support pocket.co url + SharedWithYou Analytics

## Summary
This PR
* adds support for `pocket.co` short urls
* shares the short url when available
* fixes a bug with the shared with you list
* adds SharedWithYou analytics
* Updates unit and ui tests fixtures to account for `shortUrl`

## References 
* branch identifier

## Implementation Details
* add a `shortUrl` property to `Item`
* update GraphQl queries to pull the short url
* add a GraphQl query to resolve a short url into a full url
* Add a `Route` for `pocket.co`
* Add `SharedWithYou` analytics events and updates `Home` analytics events to add SharedWithYou
* uses the `shortUrl` in share methods, when available

## Test Steps
* Have one of your contacts share one or more pocket.co urls: ideally, you should have a combination of urls containing generic urls, articles that are saved on your pocket account, syndicated articles and collections. 
* Make sure all these items appear in your shared with you list. Also make sure that the list in Home has the most recent 4 items shared, and the full list (accessible tapping on "See all") has all the items that you expect
* When tapping on a shared item from either the list or Messages, you should observe the following behavior
    - saved items and syndicated articles should open in the reader
    - collections should open in the native collection view
    - other urls should open in a web view

## PR Checklist:

## Screenshots
